addons/website/wizard/website_robots.py

```diff
@@ -7,8 +7,13 @@ class WebsiteRobots(models.TransientModel):
     _name = "website.robots"
     _description = "Robots.txt Editor"
 
-    content = fields.Text(default=lambda s: s.env['website'].get_current_website().robots_txt)
+    content = fields.Text()
+    website_id = fields.Many2one('website')
```

Comment:
The new field makes `ci/template` check red and requires special intervention from runbot/upgrade team as I remember

Reply:
Remove this new field

Reply:
I've tried, but it doesn't work if session has `force_website_id`

https://github.com/odoo/odoo/blob/d3feb26c8923cfde60894058f06dc2c2a8be05f5/addons/website/models/website.py#L568-L574

Reply:
Then pop the force website id from session in `action_open_robots` :thinking:
Or force it to selected website.. as we do when you click on go to website or install theme in settings..
`self.website_id._force()`

Reply:
And you won't need to alter the context as a bonus ;-)

Reply:
You are welcome, indeed it is ! LGTM now